spec/integration/new_spec.rb

```diff
@@ -268,25 +268,19 @@ def exec_exist?(*args)
     end
   end
 
-  context "when app directory contains a whitespace" do
-    subject(:create_app) { Open3.capture3(command) }
+  it "does not raise errors if app directory contains whitespace" do
+    app_directory = "weird dir"
+    FileUtils.mkdir_p("./tmp/#{app_directory}")
 
-    let(:app_directory) { "weird dir" }
-    let(:command) do
-      "teletype new app --no-coc --no-color --license mit --no-ext"
-    end
-
-    before { FileUtils.mkdir_p("./tmp/#{app_directory}") }
-
-    after { FileUtils.rm_r("./tmp/#{app_directory}") }
+    command = "teletype new app --no-coc --no-color --license mit --no-ext"
 
-    it "does not raise errors" do
-      within_dir("./tmp/#{app_directory}") do
-        _out, err, _status = create_app
-        puts err
-        expect(err).to eq('')
-      end
+    within_dir("./tmp/#{app_directory}") do
+      _out, err, _status = Open3.capture3(command)
+      puts err
+      expect(err).to eq("")
     end
+
+    FileUtils.rm_r("./tmp/#{app_directory}")
```

Comment:
No need for this as after the example is run the entire `tmp` directory is deleted automatically.